mazerunner/src/main/java/com/bugsnag/mazerunner/scenarios/EmptyStacktraceScenario.java

```diff
@@ -0,0 +1,19 @@
+package com.bugsnag.mazerunner.scenarios;
+
+import com.bugsnag.mazerunner.Scenario;
+
+/**
+ * Attempts to deliver a handled exception with no stacktrace.
+ */
+public class EmptyStacktraceScenario extends Scenario {
+    @Override
+    public void run() {
+        bugsnag.notify(new EmptyException("EmptyStacktraceScenario"));
+    }
+
+    private class EmptyException extends Throwable {
```

Comment:
this class could be static to avoid an implicit reference to the enclosing class

Reply:
the class is missing a serialVersionUid field

Reply:
added